test/integration/listener_extension_discovery_integration_test.cc

```diff
@@ -98,45 +119,58 @@ class ListenerExtensionDiscoveryIntegrationTest : public Grpc::GrpcClientIntegra
       filter->mutable_typed_config()->PackFrom(config);
     });
 
-    // Add an xDS cluster for extension config discovery.
-    config_helper_.addConfigModifier([](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
-      auto* ecds_cluster = bootstrap.mutable_static_resources()->add_clusters();
-      ecds_cluster->MergeFrom(bootstrap.static_resources().clusters()[0]);
-      ecds_cluster->set_name("ecds_cluster");
-      ConfigHelper::setHttp2(*ecds_cluster);
-    });
+    addEcdsCluster(std::string(EcdsClusterName));
+    // Add 2nd cluster in case of two streams.
```

Comment:
Yeah, this test is for two connections. Just updated the diff with changed  var names.  
The goal of this test is to have to ECDS filters talk to two different resources to get config update.  I guess it can be done by either two connections or the two streams in one connection with the way you mentioned in listener_lds_integration_test.cc.  Thanks for the info!